search-extension/src/main/java/com/afrigis/services/search/extension/CensusGetType.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package com.afrigis.services.search.extension;
+
+/**
+ *
+ * Report types for census
+ *
+ */
+public enum CensusGetType {
```

Comment:
Please include the service name in the enum (since it is used to select the service name elsewhere in any case).
see:
- https://docs.oracle.com/javase/tutorial/java/javaOO/enum.html